powershell-adapter/psDscAdapter/psDscAdapter.psm1

```diff
@@ -37,99 +37,197 @@ function Invoke-DscCacheRefresh {
         $Module
     )
 
-    # create a list object to store cache of Get-DscResource
-    [dscResourceCache[]]$dscResourceCache = [System.Collections.Generic.List[Object]]::new()
-
-    # improve by performance by having the option to only get details for named modules
-    # workaround for File and SignatureValidation resources that ship in Windows
-    if ($null -ne $module -and 'PSDesiredStateConfiguration' -ne $module) {
-        if ($module.gettype().name -eq 'string') {
-            $module = @($module)
+    $refreshCache = $false
+
+    $cacheFilePath = Join-Path $env:LocalAppData "dsc\PSAdapterCache.json"
```

Comment:
To improve readability, I think it's better to have it as:

```powershell
$cacheFilePath = if ($IsWindows) {
  if ($PSVersionTable.PSVersion.Major -le 5) {
    ...
  } else {
    ...
  }
} else {
  ...
}
```

Reply:
Updated.